### ISSUE TITLE
TabControl - all TabItems Headers width is always set to the widest tab

### ISSUE THREAD

**Issue by ahmed-abdelrazek** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2603]
beside #2602 using the example from the demo (changing HorizontalContentAlignment to left or right) but changing the second header to add more text will result in both tabs to have the second tab width (the widest tab)
`
<TabControl HorizontalContentAlignment="Left" materialDesign:ColorZoneAssist.Mode="SecondaryMid">
                        <TabItem Header="TAB1">
                            <TextBlock Margin="8" Text="Full-width secondary example tab 1" />
                        </TabItem>
                        <TabItem Header="TABBBBBBBBBBBBBBBBBBBB 2">
                            <TextBlock Margin="8" Text="Full-width secondary example tab 2" />
                        </TabItem>
                    </TabControl>
`
![image](https://user-images.githubusercontent.com/7858946/158997610-a06214d8-89ef-4cfc-8a78-8d49fd3a642a.png)

also it's doesn't handle overflow 
![image](https://user-images.githubusercontent.com/7858946/158997937-e658e405-918b-4eaf-83fd-4e6c58d64dd7.png)

**Comment by ahmed-abdelrazek** [comment]
changing this to stackpanel or wrappanel should fix the issue https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/24c6468b8e93211e1283ba4f4cf8765baa81486e/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TabControl.xaml#L54-L61

**Comment by veiny** [comment]
UniformGrid`s children elements are same width/height.

**Comment by ahmed-abdelrazek** [comment]
@veiny yeah that's what i found after looking into the code to track the issue changing it to any panel i suggested should fix the size thing if it was not intentional but we will still have the overflow thing to handle do we need them stacked on top of each other do we need a scrollbar martial design seems to implement hidden scrollbar https://material.io/components/tabs#interactive-demo

**Comment by MichelMichels** [comment]
I would go with the scrollbar as per spec.

As for the UniformGrid, I updated the TabControl a while ago and I can't remember if I put the UniformGrid there or if it was already present. Anyhow, the Material spec is not clear in wether this should be all the same width or not. All the examples have a same-width header so I did not think further than that. I think UniformGrid is used to achieve a full-width (accross the screen) TabControl and have the headers spaced evenly.
